src/templates/world_file.cpp

```diff
@@ -73,6 +80,31 @@ bool WorldFile::load(const QString& path)
 	return true;
 }
 
+bool WorldFile::save(const QString &path)
+{
+    if (!loaded) { return false;}
+    double parameters[6];
+    parameters[0] = pixel_to_world.m11();
+    parameters[1] = pixel_to_world.m12();
+    parameters[2] = pixel_to_world.m21();
+    parameters[3] = pixel_to_world.m22();
+    parameters[4] = pixel_to_world.m31();
+    parameters[5] = pixel_to_world.m32();
+
+    QFile file( path );
+    if ( file.open(QIODevice::WriteOnly))
+    {
+        QTextStream stream(&file);
+        for (auto& parameter : parameters)
+        {
+            stream << parameter << endl;;
+        }
```

Comment:
Double `;`.

Instead of a loop, I would prefer to have six explicit statements which access `pixel_to_world.mXX()` directly. It would be less code in total (no need to setup `parameters`), and it would express more clearly what is written to the stream.